albogdan/skule-vote
# Unit tests using real election results and data

## Overview

- Resolves #162
- A lot of finessing to make the .txt data match the format we needed


## Unit Tests Created

- 3 unit tests that use real data and results from 2022 valedictorian election, and the 2021 VPSL (in 2 instances, before and after the disqualification of Andrew Chen)


## Steps to QA

- Run the tests
- If you want to double check:
   - Valedictorian results here: https://drive.google.com/file/d/1MWpsQp0ReNUh_0wK6PBGOdEAePKSxaRH/view
   - VPSL results here: https://drive.google.com/file/d/1Iagt-MNWEpEXiA-8HkPOmj5ann5EefJ-/view
